autobuild/mac/artifacts/autobuild_mac_2_build.sh

```diff
@@ -16,4 +22,30 @@ source "$(dirname "${BASH_SOURCE[0]}")/../../ensure_THIS_JAMULUS_PROJECT_PATH.sh
 cd "${THIS_JAMULUS_PROJECT_PATH}"
 
 echo "Run deploy script..."
-sh "${THIS_JAMULUS_PROJECT_PATH}"/mac/deploy_mac.sh
+
+# If we have certificate details, then prepare the signing
+if [[ -z "${MACOS_CERTIFICATE_PWD}" ||
+     -z "${MACOS_CERTIFICATE}" ||
+     -z "${MACOS_CERTIFICATE_ID}" ||
+     -z "${NOTARIZATION_PASSWORD}" ||
+     "${SIGN}" != "sign_if_possible" ]]; then
+    sh "${THIS_JAMULUS_PROJECT_PATH}"/mac/deploy_mac.sh
```

Comment:
I‘d add a message saying "Disabled signing as not all or no credentials found" or similar.

Reply:
I didn't think this was necessary as it won't be signed 99.9% of the time.

Reply:
Yes, but better have one line extra for debugging than none.

Reply:
Seems like unnecessary debugging logging that is not needed. No message saying 'signed' = not signed. Perfectly clear.

Reply:
However: if one of the GH Secrets is not filled I‘d like to have a hint in the log. Just my personal preference…

Reply:
If one of the secrets is not filled in, then it's on me. I will be the only person using the secrets to sign.

Reply:
For now yes. But since this PR is linked in the commit history, I think someone else can look it up if needed. 
I'll accept it ;-)